runelite-client/src/main/java/net/runelite/client/plugins/worldmap/QuestStartPoint.java

```diff
@@ -30,10 +30,10 @@
 
 class QuestStartPoint extends WorldMapPoint
 {
-	QuestStartPoint(QuestStartLocation data, BufferedImage icon)
+	QuestStartPoint(QuestStartLocation data, BufferedImage icon, String toolTip)
 	{
 		super(data.getLocation(), icon);
 
-		setTooltip(data.getTooltip());
+		setTooltip("Quest start" + toolTip);
```

Comment:
the tooltip is derived just from `data`, so could the switch on the quest state be moved to here? Then wouldn't need to add a third parameter to this constructor.